Cargo.toml

```diff
@@ -4,7 +4,7 @@ name    = "sdl2"
 description = "SDL2 bindings for Rust"
 repository = "https://github.com/AngryLawyer/rust-sdl2"
 documentation = "http://angrylawyer.github.io/rust-sdl2/sdl2/"
-version = "0.30.0"
+version = "0.30.1"
```

Comment:
Please update to 0.31, or don't update sdl2 at all. There has been some breaking changes since 0.30, and if we want to update it has to be 0.31 and not 0.30.1

However I still have some changes to do before 0.31, so I'd appreciate if you could just bump sdl2-sys to 0.30.1 and leave sdl2 as-is.